packages/server/jiraImagesHandler.ts

```diff
@@ -0,0 +1,34 @@
+import {HttpRequest, HttpResponse} from 'uWebSockets.js'
+import mime from 'mime-types'
+import sleep from '../client/utils/sleep'
+import uWSAsyncHandler from './graphql/uWSAsyncHandler'
+import {NO_IMAGE_BUFFER} from './utils/atlassian/jiraImages'
+import getRedis from './utils/getRedis'
+
+const getImageFromCache = async (fileName: string, tryAgain: boolean) => {
+  const redis = getRedis()
+  const imageBuffer = await redis.getBuffer(fileName)
+  if (imageBuffer === null || imageBuffer.length === 0) return NO_IMAGE_BUFFER
+  if (imageBuffer.length > 1) return imageBuffer
+  if (tryAgain) {
+    await sleep(500)
+    return getImageFromCache(fileName, false)
+  }
+  return NO_IMAGE_BUFFER
+}
+
+const jiraImagesHandler = uWSAsyncHandler(async (res: HttpResponse, req: HttpRequest) => {
+  const fileName = req.getParameter(0)
```

Comment:
let's first validate fileName before we perform some async work:
- make sure it's not an empty string
- get the mimeType
- make sure the mimeType is an image. if it's not, return a placeholder image
- now that we know the fileName at least looks like an image's file name, call getImageFromCache